templates/hooks/cluster/sdk_file_end.go.tpl

```diff
@@ -0,0 +1,26 @@
+{{ $CRD := .CRD }}
+{{ $SDKAPI := .SDKAPI }}
+
+{{- $updateClusterVersion := (index $SDKAPI.API.Operations "UpdateClusterConfig") -}}
+
+{{/* Find the structure field within the operation */}}
+{{- range $memberRefName, $memberRef := $updateClusterVersion.InputRef.Shape.MemberRefs -}}
+{{- if (or (eq $memberRefName "Logging") (eq $memberRefName "ResourcesVpcConfig")) }}
+
+// new{{ $memberRefName }} returns a {{ $memberRefName }} object 
+// with each the field set by the resource's corresponding spec field.
+func (rm *resourceManager) new{{ $memberRefName }}(
+    r *resource,
+) *svcsdk.{{ $memberRef.ShapeName }} {
```

Comment:
and here, instead of `$memberRef.ShapeName` you would use `$field.ShapeRef.ShapeName`